compiler/qsc_frontend/src/lower.rs

```diff
@@ -443,6 +443,7 @@ impl With<'_> {
                     None
                 }
             },
+            Ok(hir::Attr::Test) => Some(hir::Attr::Test),
```

Comment:
This case is missing the validation that no arguments are provided to the attribute by checking for an empty tuple.